tests/dev/sa_error/Workflow_20200316_172120/log-info.txt

```diff
@@ -0,0 +1,26 @@
+16/03/2020 - 17:21:20 : User: ahamadeh 
+16/03/2020 - 17:21:20 : Login: ahamadeh 
+16/03/2020 - 17:21:20 : System is  validated 
+16/03/2020 - 17:21:20 : System versions: 
+16/03/2020 - 17:21:20 : R version: R version 3.6.1 (2019-07-05) 
+16/03/2020 - 17:21:20 : OSP Suite Package version: 0.1.0 
+16/03/2020 - 17:21:20 : OSP Reporting Engine version: 0.1.0 
+16/03/2020 - 17:21:20 : tlf version: 0.1.0 
+16/03/2020 - 17:21:21 : Starting run of mean model workflow
+16/03/2020 - 17:21:21 : Starting Simulate mean model
+16/03/2020 - 17:21:21 : Run simulation: LarsonSim
+16/03/2020 - 17:21:29 : Run simulation: RaltegravirSim
+16/03/2020 - 17:21:36 : Starting Calculate mean model PK parameters
+16/03/2020 - 17:21:36 : Simulation: LarsonSim
+16/03/2020 - 17:21:36 : Calculate PK parameters: C_max, C_max_norm, C_max_tD1_tD2, C_max_tD1_tD2_norm, C_max_tDLast_tEnd, C_max_tDLast_tEnd_norm, t_max, t_max_tD1_tD2, t_max_tDLast_tEnd, C_trough_tD2, C_trough_tDLast, C_tEnd, AUC_tEnd, AUC_tEnd_norm, AUC_inf, AUC_inf_norm, AUC_tD1_tD2, AUC_tD1_tD2_norm, AUC_tDLast_minus_1_tDLast, AUC_tDLast_minus_1_tDLast_norm, AUC_inf_tD1, AUC_inf_tD1_norm, AUC_inf_tDLast, AUC_inf_tDLast_norm, MRT, Thalf, Thalf_tDLast_tEnd, FractionAucLastToInf, CL, Vss, Vd
+16/03/2020 - 17:21:39 : Simulation: RaltegravirSim
+16/03/2020 - 17:21:39 : Calculate PK parameters: C_max, C_max_norm, C_max_tD1_tD2, C_max_tD1_tD2_norm, C_max_tDLast_tEnd, C_max_tDLast_tEnd_norm, t_max, t_max_tD1_tD2, t_max_tDLast_tEnd, C_trough_tD2, C_trough_tDLast, C_tEnd, AUC_tEnd, AUC_tEnd_norm, AUC_inf, AUC_inf_norm, AUC_tD1_tD2, AUC_tD1_tD2_norm, AUC_tDLast_minus_1_tDLast, AUC_tDLast_minus_1_tDLast_norm, AUC_inf_tD1, AUC_inf_tD1_norm, AUC_inf_tDLast, AUC_inf_tDLast_norm, MRT, Thalf, Thalf_tDLast_tEnd, FractionAucLastToInf, CL, Vss, Vd
+16/03/2020 - 17:21:41 : Starting Sensitivity analysis for mean model
+16/03/2020 - 17:21:41 : Simulation: LarsonSim
+16/03/2020 - 17:21:41 : Calculate sensitivity for the PK parameters: C_max, C_max_norm, C_max_tD1_tD2, C_max_tD1_tD2_norm, C_max_tDLast_tEnd, C_max_tDLast_tEnd_norm, t_max, t_max_tD1_tD2, t_max_tDLast_tEnd, C_trough_tD2, C_trough_tDLast, C_tEnd, AUC_tEnd, AUC_tEnd_norm, AUC_inf, AUC_inf_norm, AUC_tD1_tD2, AUC_tD1_tD2_norm, AUC_tDLast_minus_1_tDLast, AUC_tDLast_minus_1_tDLast_norm, AUC_inf_tD1, AUC_inf_tD1_norm, AUC_inf_tDLast, AUC_inf_tDLast_norm, MRT, Thalf, Thalf_tDLast_tEnd, FractionAucLastToInf, CL, Vss, Vd
+16/03/2020 - 17:21:57 : Simulation: RaltegravirSim
+16/03/2020 - 17:21:57 : Calculate sensitivity for the PK parameters: C_max, C_max_norm, C_max_tD1_tD2, C_max_tD1_tD2_norm, C_max_tDLast_tEnd, C_max_tDLast_tEnd_norm, t_max, t_max_tD1_tD2, t_max_tDLast_tEnd, C_trough_tD2, C_trough_tDLast, C_tEnd, AUC_tEnd, AUC_tEnd_norm, AUC_inf, AUC_inf_norm, AUC_tD1_tD2, AUC_tD1_tD2_norm, AUC_tDLast_minus_1_tDLast, AUC_tDLast_minus_1_tDLast_norm, AUC_inf_tD1, AUC_inf_tD1_norm, AUC_inf_tDLast, AUC_inf_tDLast_norm, MRT, Thalf, Thalf_tDLast_tEnd, FractionAucLastToInf, CL, Vss, Vd
```

Comment:
@abdullahhamadeh FYI: Not all PK PArameter will always be availalbe. This depends on the simualation type. For example FractionAucLastToInf is calculated for Single dosing only. C_max_tD1_tD2 for multiple dosing etc..

Maybe this should be a property of another object (for example the simulation results of
```
results <- runSensitivityAnalysis(sensitivity, sensitivityAnalysisOptions)
```

?

Reply:
Need an issue for this